src/OpenTelemetry.Instrumentation.AspNetCore/README.md

```diff
@@ -6,10 +6,21 @@
 This is an [Instrumentation
 Library](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/glossary.md#instrumentation-library),
 which instruments [ASP.NET Core](https://docs.microsoft.com/aspnet/core) and
-collect telemetry about incoming web requests.
-This instrumentation also collects incoming gRPC requests using
+collect metrics and traces about incoming web requests. This instrumentation
+also collects traces from incoming gRPC requests using
 [Grpc.AspNetCore](https://www.nuget.org/packages/Grpc.AspNetCore).
 
+**Note: This component is based on the OpenTelemetry semantic conventions for
+[metrics](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions)
+and
+[traces](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/trace/semantic_conventions),
+which are
+[experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/document-status.md),
+and hence, this package is a [pre-release](../../VERSIONING.md#pre-releases).
+Until a stable version is released, there can be breaking changes. You can track
```

Comment:
Consider adding a link to https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/telemetry-stability.md.